docs/styles/app.scss

```diff
@@ -1 +1,9 @@
 @import "node_modules/@lux-org/sass/styles/app";
+
+body.light {
+  @import "node_modules/prism-themes/themes/prism-one-light";
```

Comment:
Interesting — how does this work? You import this and then it applies themes if `body.light` applies?

Reply:
Yeah! From what I read `@import` can be done nested, so this will produce CSS that’s like: 

```css
body.light {
    .prism .token {
        /* */
    }
}

body.dark {
    .prism .token {
        /* */
    }
}
```

So we do ship two Prism themes (although they’re [not too big](https://unpkg.com/prism-themes@1.9.0/themes/prism-one-dark.min.css)), but get clean switching between them based on the class on the body and the body’s class remains the source of truth.

Reply:
Intersting, cool!